src/Chapter1.hs

```diff
@@ -614,9 +623,13 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = let last2 = mod (div (abs n) 10) 10
+                 last1 = mod (abs n) 10
```

Comment:
Great one!
You can see that this `last1` is actually the `lastDigit` function from the previous task.

Also, I notice that you use

```haskell
mod (abs n) 10
```
and 
```haskell
div (abs n) 10
```

One hint to make your solution shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(a, b) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂